README.md

```diff
@@ -68,6 +67,7 @@ This action can be configured to authenticate with GitHub App Installation or Pe
 | `EXEMPT_REPOS`            | false    |            | Comma separated list of repositories to exempt from being flagged as stale. Supports Unix shell-style wildcards. ie. `EXEMPT_REPOS = "stale-repos,test-repo,conf-*"` |
 | `EXEMPT_TOPICS`           | false    |            | Comma separated list of topics to exempt from being flagged as stale                                                                                                 |
 | `ORGANIZATION`            | false    |            | The organization to scan for stale repositories. If no organization is provided, this tool will search through repositories owned by the GH_TOKEN owner              |
+| `ADDITIONAL_METRICS`      | false    |            | Configure additional metrics like days since last release or days since last pull request. This allows for more detailed reporting on repository activity. To include both metrics, set `ADDITIONAL_METRICS: "release,pr"` |
```

Comment:
do we need to emphasize UTC? wdyt?

Reply:
My understanding was that it would be an apples to apples comparison of the utc date now and the utc date returned by the api endpoints. Which in my head meant that "days since" should always be accurate. Does that sound right? I'm happy to take another pass at it too and dig deeper into some examples.